styles/base/_typography.scss

```diff
@@ -8,79 +8,61 @@ body {
 
 .hero {
   &__tag {
+    @include font-overline;
+
     padding: 0;
     margin: 0;
-
-    &--home {
-      @include font-overline;
-
-      color: var(--pueblo);
-    }
-
-    &--child {
-      @include font-overline;
-
-      color: var(--pueblo);
-    }
+    color: var(--pueblo);
   }
 
   &__title {
+    @include font-h3;
+
     padding: 0;
     margin: 0;
+    color: var(--tuscany);
 
-    &--home {
-      @include font-h3;
+    &--profile {
+      font-weight: 300;
 
-      color: var(--tuscany);
-    }
-
-    &--child {
-      @include font-h3;
-
-      color: var(--tuscany);
+      b {
+        background-color: var(--pink);
```

Comment:
The default weight on `b` = `bold` then here could have a `font-weight: 300` like his parent